tactics/inv.ml

```diff
@@ -116,7 +116,11 @@ let make_inv_predicate env evd indf realargs id status concl =
   (* Now, we can recurse down this list, for each ai,(mkRel k) whether to
      push <Ai>(mkRel k)=ai (when   Ai is closed).
    In any case, we carry along the rest of pairs *)
-  let eqdata = Coqlib.build_coq_eq_data () in
+  let eqdata =
```

Comment:
I think it's fragile to rely on the use of exceptions to check for the registration of this data, especially if we ever want to fully implement https://github.com/coq/coq/pull/11006. The most backwards compatible way I can think of is to export a mem function in Coqlib, but if you're willing to be slightly more disruptive I'd recommend returning an option there.

Reply:
For 8.2.1, I would prefer not to be disruptive and to stay as it is, but, depending on @ejgallego's objectives wrt #11006, I can also prepare an alternative PR to #11006 which returns an option.

Reply:
It really seems that you always want to alpha-convert "8.12" into "8.2" @herbelin :laughing:

Reply:
It seems so indeed... Let me fix the comment.